api/src/auth/login_gov_jwt_auth.py

```diff
@@ -164,7 +164,9 @@ def get_final_redirect_uri(
     return f"{config.login_final_destination}?{encoded_params}"
 
 
-def validate_token(token: str, config: LoginGovConfig) -> LoginGovUser:
+def validate_token(token: str, config: LoginGovConfig | None = None) -> LoginGovUser:
+    if not config:
+        config = get_config()
 
     # TODO - this iteration approach won't be necessary if the JWT we get
     #        from login.gov does actually set the KID in the header
```

Comment:
Yeah, I wasn't sure if it would exist because the docs never mentioned it, so I built the code with the assumption it wouldn't.

After my testing yesterday, I know it does exist, so made a ticket for a future sprint to use the KID directly